packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -5357,6 +5357,7 @@ void main() {
       'TextInput.setEditingState',
       'TextInput.setEditingState',
       'TextInput.show',
+      'TextInput.show',
```

Comment:
and additional show is called due to selection change which was not called before, this should be a bug.